web/impact/impact/tests/test_accelerator_routes.py

```diff
@@ -0,0 +1,174 @@
+            'app': 'accelerator',
+            "model": "startup",
+        }
+        self.response_401(self.get(url_name, **view_kwargs))
+        startup_add_permission, _ = Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='add_startup',
+        )
+        perm_user = self.make_user(
+            'perm_user2@test.com', perms=["mc.add_startup"])
+        perm = PermissionFactory.create(codename='change_startup')
+        view_perm = PermissionFactory.create(codename='view_startup')
+        startup_permission, _ = Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='view_startup',
+        )
+        perm_user.user_permissions.add(perm)
+        perm_user.user_permissions.add(startup_add_permission)
+        perm_user.user_permissions.add(startup_permission)
+        perm_user.user_permissions.add(view_perm)
+        perm_user.save()
+        org = OrganizationFactory()
+        industry = IndustryFactory()
+        self.assertFalse(
+            Startup.objects.filter(organization=org).exists())
+        with self.login(perm_user):
+            self.post(url_name, data={
+                "is_visible": False,
+                "full_elevator_pitch": "test",
+                "linked_in_url": "",
+                "short_pitch": "test",
+                "facebook_url": "",
+                "video_elevator_pitch_url": "",
+                "organization": org.id,
+                "user": perm_user.id,
+                "primary_industry": industry.id
+            }, **view_kwargs)
+            self.assertTrue(
+                Startup.objects.filter(organization=org).exists())
+
+    def test_api_object_list(self):
+        StartupFactory(is_visible=1)
+        StartupFactory(is_visible=1)
+        StartupFactory(is_visible=1)
+        url_name = "object-list"
+        view_kwargs = {'app': 'accelerator', "model": "startup"}
+        self.response_401(self.get(url_name, **view_kwargs))
+
+        basic_user = self.make_user('basic_user@test.com')
+        with self.login(basic_user):
+            self.response_403(self.get(url_name, **view_kwargs))
+        startup_permission, _ = Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='view_startup',
+        )
+        perm_user = self.make_user(
+            'perm_user2@test.com', perms=["mc.view_startup"])
+        perm = PermissionFactory.create(codename='change_startup')
+        view_perm = PermissionFactory.create(codename='view_startup')
+        perm_user.user_permissions.add(perm)
+        perm_user.user_permissions.add(startup_permission)
+        perm_user.user_permissions.add(view_perm)
+        perm_user.save()
```

Comment:
seems like there's an entire section here that's repeated in the test above, and can be extracted and reused (and may also be abstracted better, since there're a lot of internal organs flying around at the moment.)

Reply:
refactored into its own utility method and got rid of repetition